sample-cap/db/foundation-models/azure-openai-chat-completions-entity.cds

```diff
@@ -0,0 +1,4 @@
+type Message {
```

Comment:
why is this under the db directory? Unless we want to actually store messages in a DB, I'd consider moving it to the service definition

Reply:
True, I totally forgot this. Moved.